src/types/array.rs

```diff
@@ -406,10 +406,10 @@ fn build_drop<'ctx>(
 pub fn calc_data_prefix_offset(layout: Layout) -> usize {
     get_integer_layout(32)
         .extend(get_integer_layout(32))
-        .unwrap()
+        .expect("creating a layout of two i32 will never fail")
```

Comment:
Following the Recommended Messaging Style from [Rust docs](https://doc.rust-lang.org/std/result/enum.Result.html#recommended-message-style):

```suggestion
        .expect("creating a layout of two i32 should never fail")
```